compiler/rustc_attr_data_structures/src/attributes.rs

```diff
@@ -183,17 +183,25 @@ pub enum AttributeKind {
     ConstStabilityIndirect,
 
     /// Represents [`#[deprecated]`](https://doc.rust-lang.org/stable/reference/attributes/diagnostics.html#the-deprecated-attribute).
-    Deprecation { deprecation: Deprecation, span: Span },
+    Deprecation {
+        deprecation: Deprecation,
+        span: Span,
+    },
 
     /// Represents [`#[doc]`](https://doc.rust-lang.org/stable/rustdoc/write-documentation/the-doc-attribute.html).
-    DocComment { style: AttrStyle, kind: CommentKind, span: Span, comment: Symbol },
+    DocComment {
+        style: AttrStyle,
+        kind: CommentKind,
+        span: Span,
+        comment: Symbol,
+    },
 
     /// Represents `#[rustc_macro_transparency]`.
```

Comment:
This doc-comment is now on the wrong place

Reply:
I just noticed myself when editing another commit hehe, wanted to push and then saw your comment as well. Goot catch!